<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentUser</class>
 <widget class="QDialog" name="StudentUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生页面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#tool QPushButton{border:none;background-color:rgb(84,92,100);min-height:55px;font-size:14px;font-family:&quot;黑体&quot;;color:white;}
QWidget#tool QPushButton:checked{border:none;background-color:rgb(67,74,80);}
QWidget#tool {background-color:rgb(84,92,100);}
QDialog{background-color:white;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="tool" native="true">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <widget class="QPushButton" name="btn_pa">
        <property name="text">
         <string>奖惩情况</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_course">
        <property name="text">
         <string>选课系统</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>117</width>
          <height>37</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn_grade">
        <property name="text">
         <string>成绩管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_info">
        <property name="text">
         <string>个人信息</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QPushButton[name=&quot;btn&quot;]{font-size:14px;min-height:28px;padding:4px;min-width:85px}
QLineEdit[name=&quot;le&quot;]{border-radius:4px;min-height:25px;border:1px solid gray;min-width:280px;max-width:280px;}
QLabel[name=&quot;lb&quot;]{font:14px '黑体';}
QLabel[name=&quot;title&quot;]{font:18px '宋体';}
QWidget[name=&quot;bg&quot;]{background-color:white;}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
